Doc/library/stdtypes.rst

```diff
@@ -1464,6 +1566,7 @@ multiple fragments.
    sections.  In addition, see the :ref:`stringservices` section.
 
 
+
```

Comment:
Was this change intentional?

Reply:
Thanks for catching that!  I've removed it.